src/main/java/ch/kalunight/zoe/command/create/CreatePlayerCommand.java

```diff
@@ -44,40 +41,42 @@ public CreatePlayerCommand() {
   }
 
   @Override
-  public void executeCommand(CommandEvent event) {
+  public void executeCommand(CommandEvent event) throws SQLException {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'executeCommand(CommandEvent)' has an NPath complexity of 6912](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=4739924)